requirements.txt

```diff
@@ -3,3 +3,4 @@ django-model-utils
 swapper
 djangorestframework>=3.7,<3.8
 passlib
+openwisp-utils
```

Comment:
Also mentioning the versions is a good practice. Helps you handle compatibility.